build/mac/build_delta_installer.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import optparse
+import os
+import re
+import subprocess
+import sys
+
+
+def unmount(volume_path):
+  print('-> unmount ' + volume_path)
+  command = ['hdiutil', 'detach', volume_path]
+  try:
+      subprocess.check_call(command)
+  except subprocess.CalledProcessError as e:
+      print(e.output)
+      raise e
+
+
+def mount_dmg(dmg_path, mount_point):
+  print('-> mounting ' + dmg_path + ' to ' + mount_point)
+
+  command = ['hdiutil', 'attach', dmg_path, '-mountpoint', mount_point]
+  try:
+      output = subprocess.check_output(command)
+  except subprocess.CalledProcessError as e:
+      print(e.output)
+      raise e
+
+
+def Main(argv):
+  parser = optparse.OptionParser('%prog [options]')
+  parser.add_option('--binary-delta', dest='binary_delta_path', action='store',
+      type='string', default=None, help='The path of BinaryDelta binary.')
+  parser.add_option('--sign-update', dest='sign_update_path', action='store',
```

Comment:
this goes out once signing is extracted to GN

Reply:
removed.